src/workspaces/common/state/useWorkspace.ts

```diff
@@ -145,7 +145,12 @@ export const useWorkspace = (namespace, name): WorkspaceDetails => {
       storageDetails.fetchedLocation = 'SUCCESS';
       setGoogleStorage(storageDetails);
     } catch (error) {
-      setGoogleStorage({ fetchedLocation: 'ERROR', location, locationType });
+      const errorText = await error.text();
```

Comment:
TODO: Checking here too causes some tests to fail because the error is read twice. Have not fixed this yet.